INPTPZ1/ValueObjects/Polynome.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace INPTPZ1.ValueObjects
+{
+    class Polynome
+    {
+        private List<ComplexNumber> ComplexNumbers { get; set; }
+
+        public Polynome()
+        {
+            ComplexNumbers = new List<ComplexNumber>();
+        }
+
+        public Polynome(List<ComplexNumber> complexNumbers)
+        {
+            ComplexNumbers = complexNumbers;
+        }
+
+        public Polynome Derive()
+        {
+            List<ComplexNumber> roots = new List<ComplexNumber>();
+
+            for (int i = 1; i < ComplexNumbers.Count; i++)
+            {
+                roots.Add(ComplexNumbers[i].Multiply(new ComplexNumber() { Re = i }));
+            }
+
+            return new Polynome(roots);
+        }
+        
+        public ComplexNumber Eval(ComplexNumber x)
+        {
+            ComplexNumber s = ComplexNumber.Zero;
```

Comment:
s, coef, bx - are these names some cryptic mumbling?

Reply:
Well, there is a lot of changes I´d like to do. But to be honest, in my view this topic(Newton´s Fractals) si not easy to understand. I wanted to changes these names, but I don´t know what these variables mean. As well as method name. Can I ask you for deeper explanation? Thank you.